proposals/GithubMigration.md

```diff
@@ -0,0 +1,58 @@
+# Migration of Jira Issues to Github
+- Status: Proposed
+
+## Overview
+This is a proposal to migrate Jira issues over to GitHub. [See issue CB-13157](https://issues.apache.org/jira/browse/CB-13157)
+
+## Current
+Currently, issues for all Cordova repos are tracked on [Jira](https://issues.apache.org/jira/secure/Dashboard.jspa).
+
+## Proposal
+This proposal recommends that issues are mirgated from Jira to Github so that the issues are with the code. This is recommended because:
+* Jira can be a barrier to actually creating issues.
+* This is an expected open source workflow.
+* This helps to modernize Cordova.
+* Issues should be with the code.
+
+There are two potential options when using Github to track issues:
+
+## Option 1
+When issues are migrated to Github, each repository will contain its own set of issues. In each repo, there will be a list of the issues, which can be found under the Github issue tab. Therefore, the issues can be found with the code. This can allow developers to reference pull requests more easily. Developers will also be able to "watch" repos and receive notifications when pull requests are sent or when new issues are created. We encourage and want to make it easy for committers to watch Github for the repos they want to be involved in.
+
+## Option 2
+When issues are migrated to Github, they are added to one "issues" repository. There will be one master “issues” repo that can track all of the Cordova issues on GitHub.
+
+## Notes
+In both of these scenarios, issues are migrated over from Jira to Github. Jira will continue to be around for security issues. Alternatively, we could use a private GitHub repo for only pmc members.
```

Comment:
Alternatively, we could use a private GitHub repo for security tracking (if private repos are an option on the github apache org).